client/src/actions/teams.js

```diff
@@ -103,3 +104,203 @@ export const createTeam = (name, isPublic) => (dispatch) => {
             dispatch(hideLoadingModal());
         });
 };
+// ===== Change team visibility ===== //
+export const CHANGE_VISIBILITY_TEAM_STARTED = 'teams/CHANGE_VISIBILITY_TEAM_STARTED';
+export const CHANGE_VISIBILITY_TEAM_FAILURE = 'teams/CHANGE_VISIBILITY_TEAM_FAILURE';
+export const CHANGE_VISIBILITY_TEAM_SUCCESS = 'teams/CHANGE_VISIBILITY_TEAM_SUCCESS';
+
+export const changeVisibilityTeamStartedAction = () => ({ type: CHANGE_VISIBILITY_TEAM_STARTED });
+
+export const changeVisibilityTeamFailureAction = () => ({ type: CHANGE_VISIBILITY_TEAM_FAILURE });
+
+export const changeVisibilityTeamSuccessAction = isVisible => ({
+    type: CHANGE_VISIBILITY_TEAM_SUCCESS,
+    payload: {
+        isVisible,
+    },
+});
+
+export const changeVisibility = (teamId, visibility) => (dispatch) => {
+    dispatch(changeVisibilityTeamStartedAction());
+    dispatch(displayLoadingModal());
+    const resource = 'teams'.concat(`/${teamId}/visibility`);
+    APIFetch.fetchPrelloAPI(resource, { isVisible: !visibility }, APIFetch.PUT)
```

Comment:
J'aime pas, c'est pas intuitif de filer une visibilité pour avoir l'inverse ^^

Et c'est jamais une bonne chose que de switch un booléen, vaut mieux spécifier la valeur de ce qu'on veut, c'est plus fiable (quitte à ce que le container envoie !this.props.visibility par exemple mais faut que la roue put avec la valeur qu'on lui file).